test/XRP/issued-currency-client-integration.test.ts

```diff
@@ -417,4 +420,395 @@ describe('IssuedCurrencyClient Integration Tests', function (): void {
     assert.equal(createdTrustLine.limit, trustLineLimit)
     assert.equal(createdTrustLine.currency, trustLineCurrency)
   })
+
+  // TODO: (acorso) can any addresses be created in `before` and reused?  Can any test cases be combined?
```

Comment:
it's good to have tests be simple and focused if you find yourself trying to figure out how to given/when/then it, your test is probably doing too much